Assets/Resources/JSON/Languages.json

```diff
@@ -0,0 +1,243 @@
+{
+  "Lookup" : [
```

Comment:
Do you find a single JSON file would work better than one per language? I think Ira had it the other way. I'm just wondering in terms of scalability. Say we hire a translator for each language: it would make more sense to give them one file to manage rather than making them scroll through a file with potentially several languages per key. My other thought is that a word might not have a direct translation in some cases, which may produce errors if they're left out here. Obviously not important right now, but just thought I'd mention it.